frontend/ui/widget/menu.lua

```diff
@@ -167,7 +171,7 @@ function MenuItem:init()
         }
     end
 
-    local mandatory = self.mandatory and ""..self.mandatory.." " or ""
+    local mandatory = self.mandatory and ""..self.mandatory or ""
```

Comment:
Wouldn't self.mandatory suffice?

Reply:
which one ? :)
I think this aims at converting any self.mandatory being a number to being a string

Reply:
We'll let it be, but it's also passed in as `""..mandatory` a couple lines down.